src/routes/plans/[id]/merge/+page.ts

```diff
@@ -0,0 +1,38 @@
+import { base } from '$app/paths';
+import { redirect } from '@sveltejs/kit';
+import effects from '../../../../utilities/effects';
+import type { PageLoad } from './$types';
+
+export const load: PageLoad = async ({ params }) => {
+  const { id } = params;
```

Comment:
Can we add an auth check here (see [here](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/src/routes/plans/%5Bid%5D/%2Bpage.ts#L7) for example):

```ts
const { user } = await parent();

if (!user) {
  throw redirect(302, `${base}/login`);
}
```